activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Add dirties option to uncached
+
+    This adds a `dirties` option to `ActiveRecord::Base.uncached` and
+    `ActiveRecord::ConnectionAdapters::ConnectionPool#uncached`.
+
+    When set to `false`, database writes to the affected connection pool will not clear query caches.
+
+    *Donal McBreen*
+
```

Comment:
Would mention its motivation (to support Solid Cache), its purpose (to isolate solid cache writes from needlessly blowing query cache on other connections), and its default value (true, writes dirty others' caches by default since they may be replica conns in the same cluster).